source/server/hot_restart_impl.cc

```diff
@@ -27,7 +27,7 @@ namespace Server {
 // from working. Operations code can then cope with this and do a full restart.
 const uint64_t SharedMemory::VERSION = 8;
 
-SharedMemory& SharedMemory::initialize(Options& options) {
+SharedMemory& SharedMemory::initialize(Server::Options& options) {
```

Comment:
nit: you should be able to revert all of the `Server::` changes from these two moved files (h/cc).